sequencer/src/block.rs

```diff
@@ -72,6 +149,10 @@ pub struct Header {
 
     pub payload_commitment: VidCommitment,
     pub transactions_root: NMTRoot,
+    /// Root Commitment of Block Merkle Tree
+    pub block_merkle_tree_root: <SHA3MerkleTree as MerkleTreeScheme>::Commitment,
```

Comment:
Nit: might be good to make a type alias for this (e.g. `SHA3MerkleCommitment` or something)